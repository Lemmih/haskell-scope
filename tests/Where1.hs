module Where1 where

a = b
  where b = b
