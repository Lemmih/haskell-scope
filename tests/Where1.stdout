Scope errors:

Definitions:
  Definition 1:
    [4ma[0m = b
  Definition 2:
      where [4mb[0m = b

Use sites:
  Definition used: 2
    a = [4mb[0m
  Definition used: 2
      where b = [4mb[0m
